subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -262,9 +228,26 @@ void radio_df_ant_switch_spacing_set_4us(void)
 	radio_df_ant_switch_spacing_set(RADIO_DFECTRL1_TSWITCHSPACING_4us);
 }
 
-void radio_df_ant_switch_pattern_set(uint8_t pattern)
+void radio_df_ant_switch_pattern_set(uint8_t *patterns, uint8_t len)
```

Comment:
The PS says "A minimum number of 3 patterns must be written." -- should this return an error if `len` is too short?

Reply:
PS requires 3 patterns but pay attention that first pattern (`SWITCHPATTERN[0]`) is set with value provided by DTS. 
So there is a need for 2 more patterns and that is verified by HCI commands (according to BT spec. min. patterns number is 2). So there is no need to verify it again here. ULL handles that.

Reply:
>  that is verified by handling of HCI commands (according to BT spec. min. patterns number is 2). So there is no need to verify it again here. ULL handles that.

Great, thanks.